tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -67,9 +67,15 @@ def get_run_info(metadata_root, product, **kwargs):
 class RunInfo(dict):
     def __init__(self, metadata_root, product, debug, browser_version=None, extras=None):
         import mozinfo
-
         self._update_mozinfo(metadata_root)
         self.update(mozinfo.info)
+
+        from update.tree import GitTree
+        self._gittree = GitTree()
+        rev = self._gittree.rev
```

Comment:
Why do we create a property on `self` here? is it used elsewhere?

Reply:
Not actually... Let me change it to a local variable.